packages/web/src/components/hover-card/AudioHoverCard.tsx

```diff
@@ -69,12 +73,24 @@ const getBadgeName = (tier: BadgeTier) => {
 export const AudioHoverCard = ({
   children,
   tier,
-  amount,
+  userId,
   onClose,
   anchorOrigin,
   transformOrigin,
   onClick
 }: AudioHoverCardProps) => {
+  // Get user's balance information
+  const { data: user } = useUser(userId)
```

Comment:
since we just need total_balance, lets do this:
```
const {data: audioValue} = useUser(userId, {select: user => formatCount(number(AUDIO(BN(user.total_balance)).toFixed(2))
})
```
feel free to break up things however you want, but the core thing is using select to get "total_balance"